src/bin/nydus-image/main.rs

```diff
@@ -759,7 +759,11 @@ fn main() -> Result<()> {
             }
         }
     } else if let Some(matches) = cmd.subcommand_matches("merge") {
-        Command::merge(matches, &build_info)
+        let result = Command::merge(matches, &build_info);
+        if let Err(MergeError::InconsisentFsVersion(_)) = result {
+            std::process::exit(2);
+        }
+        Ok(())
```

Comment:
Other errors should also be handled here?

Reply:
yep